packages/waku/src/lib/renderers/html-renderer.ts

```diff
@@ -259,12 +259,14 @@ export const renderHtml = async (
           {},
           {
             get(_target, name: string) {
-              const file = filePath.slice(config.basePath.length);
+              const file =
+                '/' + filePath.slice(config.basePath.length).split('?')[0]!;
```

Comment:
it smells something.

Reply:
what's that smell?

Reply:
that it doesn't work on edge cases, might be the case with windows. not sure.
why do we need to add `/` prefix?

Reply:
to make it a vite path. let's remove it, even before that, it was failing, so this has no effect.